lib/neutron-legacy

```diff
@@ -440,7 +440,11 @@ function install_mutnauq {
 function install_neutron_agent_packages_mutnauq {
     # radvd doesn't come with the OS. Install it if the l3 service is enabled.
     if is_service_enabled q-l3; then
-        install_package radvd
+	if is_clearlinux; then
```

Comment:
Do the same thing here: ! is_clearlinux, adding openstack-common to files/rpms-clear/general covers all of these attempted installations automatically.